test/k8sT/Policies.go

```diff
@@ -282,6 +284,49 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 				helpers.DefaultNamespace, appPods[helpers.App2],
 				helpers.CurlFail("http://%s/public", clusterIP))
 			res.ExpectSuccess("%q cannot curl to %q public", appPods[helpers.App2], clusterIP)
+
+			By("ServiceAccount")
```

Comment:
I know that is sitll WIP, but please do the test in another `By()` ;-)

Reply:
I don't understand the comment. Change the text in the By()? Move it to somewhere else?

Reply:
Fixed